pkg/permissions/labelled-accounts/mainnet.json

```diff
@@ -0,0 +1,13 @@
+{
+  "0x10a19e7ee7d7f8a52822f6817de8ea18204f2e4f": "DAO_MULTISIG",
+  "0x166f54f44f271407f24aa1be415a730035637325": "BALLER_OPS_MULTISIG",
+  "0x75a52c0e32397a3fc0c052e2ceb3479802713cf4": "LINEAR_POOL_CONTROLLER_BALLER_MULTISIG",
+  "0xf4a80929163c5179ca042e1b292f5efbbe3d89e6": "SWAP_FEE_CONTROLLER_BALLER_MULTISIG",
+  "0xc38c5f97b34e175ffd35407fc91a937300e33860": "LM_MULTISIG",
+  "0x7c68c42de679ffb0f16216154c996c354cf1161b": "TREASURY_MULTISIG",
+  "0xa29f61256e948f3fb707b4b3b138c5ccb9ef9888": "EMERGENCY_SUBDAO_MULTISIG",
+  "0x02f35dA6A02017154367Bc4d47bb6c7D06C7533B": "BLABS_OPS_MULTISIG",
+  "0xd2eb7bd802a7ca68d9acd209bec4e664a9abdd7b": "BLABS_VEBAL_MULTISIG",
+  "0xe4a8ed6c1d8d048bd29a00946bfcf2db10e7923b": "GAUNTLET_FEE_SETTER",
+  "0xc92e8bdf79f0507f65a392b0ab4667716bfe0110": "GNOSIS_PROTOCOL_RELAYER"
```

Comment:
It'd be good if we could also have a short description of what these are, their scope, and potentially even link to gov discussion. We could add a metadata field, or make this a ts file and just write comments